gnc/navigator_trajectory_generation/tools/ogrid_gen.py

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env python
+import rospy
```

Comment:
This is super cool!!

In the future, you should add a command line -h option people easily know how to use it.